casepro/msg_board/views.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import unicode_literals
+
+from django.http import JsonResponse, HttpResponse
+from django.core.exceptions import PermissionDenied
+from django.utils import timezone
+
+from smartmin.views import SmartListView, SmartTemplateView
+from smartmin.views import SmartReadView, SmartCRUDL
+from dash.orgs.models import Org
+from dash.orgs.views import OrgPermsMixin, OrgObjPermsMixin
+from casepro.msg_board.models import MessageBoardComment
+
+
+class MessageBoardView(OrgPermsMixin, SmartTemplateView):
+    template_name = 'msg_board/comment_list.haml'
+    permission = 'orgs.org_inbox'
+
+
+class CommentsCRUDL(SmartCRUDL):
+    title = 'Comments'
+    actions = ('list', 'pinned', 'pin', 'unpin')
+    model = MessageBoardComment
+
+    class List(OrgPermsMixin, SmartListView):
+        permission = 'msg_board.messageboardcomment_list'
+        default_order = ('-submit_date',)
+
+        def get_queryset(self):
+            return MessageBoardComment.objects.for_model(Org).filter(
```

Comment:
I disagree. I think this would unnecessarily pollute the `Org` model. Also, since the `Org` model is defined in `dash`, the only way to add this helper method would be to patch it it, which feels unnecessary especially since we only need it in 2 places at the moment.